File: 01. SourceCode/ChildFund/InformationHub.Model/ChangePasswordUserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InformationHub.Model
{
    public class ChangePasswordUserModel
    {
        /// <summary>
        /// Id người dùng
        /// </summary>
        public string Id { get; set; }

        /// <summary>
        /// Mật khẩu cũ
        /// </summary>
        public string PasswordOld { get; set; }

        /// <summary>
        /// Mật khẩu mới
        /// </summary>
        public string PasswordNew { get; set; }

        /// <summary>
        /// Xác nhận mật khẩu mới
        /// </summary>
        public string ConfirmPasswordNew { get; set; }
    }
}

File: 01. SourceCode/Android/ChildProfile/app/src/main/java/nts/childprofile/model/ChangePasswordUserModel.java
package nts.childprofile.model;

/**
 * Created by NTS-VANVV on 09/01/2019.
 */

public class ChangePasswordUserModel {
    /// <summary>
    /// Id người dùng
    /// </summary>
    public String Id;

    /// <summary>
    /// Mật khẩu cũ
    /// </summary>
    public String PasswordOld;

    /// <summary>
    /// Mật khẩu mới
    /// </summary>
    public String PasswordNew;

    /// <summary>
    /// Xác nhận mật khẩu mới
    /// </summary>
    public String ConfirmPasswordNew;
}
